benchmarks/erp12/cbgp_lite/benchmark/suite/psb.clj

```diff
@@ -334,10 +769,10 @@
                                                                             :n-train  0})))))
             start-time (System/currentTimeMillis)
             evaluation (factory (:solution task) nil)
-            duration (/ (- (System/currentTimeMillis) start-time) 1000)]
+            duration (/ (- (System/currentTimeMillis) start-time) 1000.0)]
         (cond
           (> (:total-error evaluation) 0)
-          (throw (ex-info (str problem-name " solution has non-zero error.") {:eval evaluation}))
+          (throw (ex-info (str problem-name " solution has non-zero error." evaluation) {:eval evaluation}))
```

Comment:
The `evaluation` data is already included in the exception's data.  I believe it is visible in the full stack track and can be accessed at runtime (ie. REPL) like this:

```clojure
(try 
  (validate-solutions {:data-dir "data/psb/" :num-cases 50})
  (catch clojure.lang.ExceptionInfo e
    (ex-data e))) ;; This one returns the map {:eval evaluation}
```

Reply:
I don't remember why I made this change, but happy to undo it!